### ISSUE TITLE
Unable to import ERR_REQUIRE_ESM 

### ISSUE THREAD

**Issue by ansh** [sachinraja/trpc-playground#36]
```
Error [ERR_REQUIRE_ESM]: require() of ES Module /Developer/backend/node_modules/@trpc-playground/html/dist/index.js from /Developer/backend/node_modules/trpc-playground/dist/handlers/express.cjs not supported.
Instead change the require of index.js in /Developer/backend/node_modules/trpc-playground/dist/handlers/express.cjs to a dynamic import() which is available in all CommonJS modules.
```

Not sure what this error message means. Some Googling suggest I change "module" and "target" in `tsconfig.json` but them moment I make them `esnext`, all of my other package imports (including `trpc`) break!

NodeJS v16.18.1
TypeScript 4.9.3
"trpc-playground": "^1.0.0",
"@trpc/server": "^10.5.0",

And my tsconfig was just created by running `tsc --init`

**Comment by sachinraja** [comment]
fixed in `1.0.1`

**Comment by ansh** [comment]
beast mode!
